mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
I had a conversation with @tokunbo in person about this and realized that I did not really make my concerns here clear, so let me spell it out in more detail.

My concern is that any time Mimic produces static, fixed values as part of a mock, that is an opportunity for a test, usually accidentally, to start depending on those values.  Once that happens, if Mimic were to start producing more "correct" values, whether synthetic, injected by a control-plane mutation, or computed by a control-plane behavior, test suites would break and it would become difficult for us to change to make things more accurate.

So my major concern about landing this as-is is that we start depending on these magic values, both in Cloud Intelligence's test suite (again, probably not now, probably in a future change that *accidentally* starts depending on one value being a certain number, or within a certain range), and then we have to live with them forever, and it's hard to explain to future contributors to the cloud monitoring mock that `7.1695253641707` is a number they should care about for some reason :).

One way to address this, hopefully quickly, is to return a deterministic pattern of generated numbers for these values, in a certain range.  If we start adding real, system randomness, this could make Mimic's tests hard to read (since gathering real entropy is a non-deterministic I/O operation).